src/jobflow_remote/remote/host/remote.py

```diff
@@ -353,18 +352,15 @@ def inter_handler(title, instructions, prompt_list):
     Used by Interactive AuthSource.
     """
     if title:
-        print(title.strip())
+        pass
     if instructions:
-        print(instructions.strip())
+        pass
```

Comment:
I'm not sure that these particular print statements are useful, but probably this whole chunk can be removed if not

Reply:
oops, i missed this one. meant to revert all the `ruff` print removals. not sure if they're good to keep but that can be decided at a later time